backend/src/main/java/com/nikogrid/backend/exceptions/DuplicateUserException.java

```diff
@@ -0,0 +1,11 @@
+package com.nikogrid.backend.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.CONFLICT) // 409
```

Comment:
You don't need these annotations as the error handler is the one responsible for setting the status code.